### ISSUE TITLE
Error 409 for adding keyple-plugin-springcard-android-pcsclike-java-lib:1.0.0

### ISSUE THREAD

**Issue by salhugues** [springcard/android-keyple#4]
Hi, currently I have an issue with implementation of your library, when I try to build the project I have this error:

`Could not GET 'https://artifact.socrate.vsct.fr/artifactory/ritmx-mvn-snapshots-local/com/springcard/keyple/keyple-plugin-springcard-android-pcsclike-java-lib/1.0.0/keyple-plugin-springcard-android-pcsclike-java-lib-1.0.0.pom'. Received status code 409 from server`

Thank you for you  help

**Comment by jeanpierrefortune** [comment]
Hi,
I don't know how you publish artifacts in your system but obviously gradle is looking for the plugin in a place where it is not.
Have you kept the same gradle configuration?
Have you tried publishing commands like `gradle install`?
Keep me informed.

**Comment by salhugues** [comment]
> Hi, I don't know how you publish artifacts in your system but obviously gradle is looking for the plugin in a place where it is not. Have you kept the same gradle configuration? Have you tried publishing commands like `gradle install`? Keep me informed.

Uhm I've just followed steps described in readme.md, I added these 4 lines in build gradle:

`implementation 'org.calypsonet.terminal:calypsonet-terminal-reader-java-api:1.0.+'`
`implementation 'org.eclipse.keyple:keyple-common-java-api:2.0.+'`
`implementation 'org.eclipse.keyple:keyple-util-java-lib:2.+'`
`implementation 'org.eclipse.keyple:keyple-service-java-lib:2.0.1'`
`implementation 'com.springcard.keyple:keyple-plugin-springcard-android-pcsclike-java-lib:1.0.0'`

And it gave me the error from my first post on project compilation. I think there's something wrong with the publication of this artefact keyple-plugin-springcard-android-pcsclike-java-lib .

**Comment by jeanpierrefortune** [comment]
Indeed, the README file is in advance of the publication of the plugin on MavenCentral which is not yet effective.
In the meantime, the easiest way is to get the aar artifact locally and integrate it as a local library.

**Comment by salhugues** [comment]
> Indeed, the README file is in advance of the publication of the plugin on MavenCentral which is not yet effective. In the meantime, the easiest way is to get the aar artifact locally and integrate it as a local library.

Ok this is what I thought.

Thanks for your help.
